openmdao/core/petsc_impl.py

```diff
@@ -227,37 +227,25 @@ def get_view(self, system, comm, varmap):
         if trace: debug("petsc_vec creation DONE")
         return view
 
-    def distance_along_vector_to_limit(self, alpha, duvec, vector_alpha):
+    def distance_along_vector_to_limit(self, alpha, duvec):
```

Comment:
this function does nothing now that the base class function doesn't do, so just get rid of it.